File: cs/HAC.Tests.Unit/AlphabetSubSequence.Test.cs
using NUnit.Framework;
using NUnit.Framework.Constraints;

namespace HAC.Tests.Unit
{
    public class AlphabetSubSequence
    {
        [TestCase("effg", false)]
        [TestCase("ace", true)]
        public void ShouldWork(string data, bool expected)
        {
            Assert.AreEqual(expected, HAC.AlphabetSubsequence(data));
        }
    }
}

File: php/tests/AlphabetSubSequenceTest.php
<?php

use PHPUnit\Framework\TestCase;

require "src/alphabet_subsequence.php";

final class AlphabetSubSequenceTest extends TestCase
{
    public function testAlphabetSubSequence()
    {
        $data = "effg";
        $actual = alphabet_subsequence($data);
        $expected = false;
        $this->assertEquals($expected, $actual);
    }

    public function testAlphabetSubSequence2()
    {
        $data = "ace";
        $actual = alphabet_subsequence($data);
        $expected = true;
        $this->assertEquals($expected, $actual);
    }
}